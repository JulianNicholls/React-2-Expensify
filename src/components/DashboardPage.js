import React from 'react';

import ExpenseList from './ExpenseList';

const DashboardPage = () => (
  <div className="container">
    <h1>Dashboard</h1>
    <ExpenseList />
  </div>
);

export default DashboardPage;
