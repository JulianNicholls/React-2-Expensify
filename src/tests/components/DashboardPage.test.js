import React from 'react';
import { shallow } from 'enzyme';

import DashboardPage from '../../components/DashboardPage';

describe('DashboardPage component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<DashboardPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
