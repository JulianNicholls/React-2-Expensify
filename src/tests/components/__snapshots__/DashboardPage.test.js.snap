// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardPage component should render correctly 1`] = `
<div
  className="container"
>
  <h1>
    Dashboard
  </h1>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
